scripts/superdesk-search/search.js

```diff
@@ -2020,10 +2020,9 @@
                         var listComponent = ReactDOM.render(itemList, elem[0]);
 
                         $document.on('keydown', function(evt) {
-                            if (evt.target.className.match('-view list-view') != null) {
```

Comment:
@sjunaid this prevents keyboard controlling in content lists - you have to put focus on dropdown and stop event propagation instead